PennMobile/Events/Views/PennEventCellView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  PennEventCellView.swift
+//  PennMobile
+//
+//  Created by Jacky on 3/3/24.
+//  Copyright © 2024 PennLabs. All rights reserved.
+//
+
+import SwiftUI
+
+struct PennEventCellView: View {
+    @ObservedObject var viewModel: PennEventViewModel
+    
+    let screenWidth = UIScreen.main.bounds.width
+    
+    @Environment(\.colorScheme) var colorScheme
+    
+    var shadowColor: Color {
+        colorScheme == .dark ? .white.opacity(0.2) : .black.opacity(0.2)
+    }
+    
+    var body: some View {
+        ZStack {
+            if let imageUrl = viewModel.imageUrl {
+                AsyncImage(url: imageUrl) { phase in
+                    if let image = phase.image {
+                        image.resizable()
+                             .aspectRatio(contentMode: .fill)
+                    } else if phase.error != nil {
+                        Image("pennmobile")                             
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                    } else {
+                        ProgressView()
+                    }
+                }
+                .frame(width: screenWidth - 30, height: 190)
```

Comment:
Probably stay away from using `UIScreen.main.bounds.width`. Use a `GeometryReader` instead and get the width that way (so it's relative to the parent view)